.github/workflows/phpunit.yml

```diff
@@ -76,20 +79,15 @@ jobs:
       - name: "Install highest dependencies"
         if: ${{ matrix.dependencies == 'highest' }}
         run: |
-          composer update --no-interaction --no-progress --no-suggest
+          composer install --no-interaction --no-progress --no-suggest
```

Comment:
Shouldn't this be `update` instead of `install`?
AFAIK `install` is to use the current `composer.lock` file, whereas `update` actually updates the packages

Reply:
yeah with composer v2 i dont actually need the run twice for merge plugin either